python/hidet/graph/ops/definitions/conv3d_transpose/conv3d_transpose.py

```diff
@@ -0,0 +1,112 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Sequence, Union, Tuple
+from hidet.ir.expr import if_then_else, LogicalAnd
+from hidet.ir.compute import compute, reduce
+from hidet.graph.ops.definitions.utils import Task, Operator, Tensor, TensorNode
+from hidet.graph.ops.definitions.utils import input_like, normalize_stride, normalize_padding
+
+
+class Conv3dTransposeTask(Task):
+    def __init__(
+        self,
+        data: TensorNode,
+        weight: TensorNode,
+        stride: Tuple[int, int, int],
+        padding: Tuple[int, int, int, int, int, int],
+        groups: int,
+        output_padding: Tuple[int, int, int],
+    ):
+        n, oc, r, p, q = data.const_shape()
+        oc, wc, kz, kx, ky = weight.const_shape()
+        c = wc * groups
+        sz, sx, sy = stride
+        pz0, px0, py0, pz1, px1, py1 = padding
+        z = (r - 1) * sz + -pz0 - pz1 + kz + output_padding[0]
```

Comment:
Consider avoid writing the expression like `a + -b` which is not human-readable.